### DISCUSSION TITLE
Wiki pages contain IP grabbers

### DISCUSSION THREAD

**Discussion by amywarble** [AvaloniaUI/Avalonia#8376] [General]
The "Projects that are using Avalonia" wiki page contains IP grabber links using "linkify.me".  These were added on December 18, 2021 in commit 16da225.  [Diff comparison](https://github.com/AvaloniaUI/Avalonia/wiki/Projects-that-are-using-Avalonia/_compare/e7e133de234974cd775c37258680a7a49b5e9fbe...16da2257c5e0cf4d072ab0e0b49904721ee513e7)

The "Remotely debugging AvaloniaUI on Linux OSX" page also contains a IP grabber link.  These were added on December 18, 2021 in commit f22aecc.  [Diff comparison](https://github.com/AvaloniaUI/Avalonia/wiki/Remotely-debugging-AvaloniaUI-on-Linux-OSX/_compare/c4533b0a10f55b18d16fd4455cc1cc5aeca7c37b...f22aeccae531e3a725ad6d0879af2e14967debda)

I can't figure out how to remove these myself.

**Comment by maxkatz6** [comment]
Thank you!
It seems like this deleted user https://github.com/irinl5 edited the page.
Probably it was edited before we restricted access to it.

**Reply by maxkatz6** [reply]
I fixed these link

**Comment by amywarble** [comment]
No problem, @maxkatz6.  👍
